contentcuration/contentcuration/viewsets/contentnode.py

```diff
@@ -776,15 +777,19 @@ def size(self, request, pk=None):
             # When stale, that means the value is not up-to-date with modified files in the DB,
             # and the channel is significantly large, so we'll queue an async task for calculation.
             # We don't really need more than one queued async calculation task, so we use
-            # get_or_create_async_task to ensure a task is queued, as well as return info about it
+            # fetch_or_enqueue to ensure a task is queued, as well as return info about it
             task_args = dict(node_id=node.pk, channel_id=node.channel_id)
-            get_or_create_async_task(
-                "calculate-resource-size", self.request.user, **task_args
-            )
+            task_result = calculate_resource_size_task.fetch_or_enqueue(self.request.user, **task_args)
+
+        changes = []
```

Comment:
The change propagation to the frontend _should_ be handled by the sync endpoint polling now - so I think we can probably remove this?

Reply:
Ah thanks, I meant to clean this up. :+1: